libs/crucible_proc_macros/src/lib.rs

```diff
@@ -52,33 +52,42 @@ impl Folder {
     fn handle_call(&mut self, call: ExprCall) -> Expr {
         // Check whether this is a call to the subject function.
         match *call.func {
-            Expr::Path(ref x) if x.path == self.subject_function_path => {},
+            Expr::Path(ref x) if x.path == self.subject_function_path => {}
             _ => return fold::fold_expr_call(self, call).into(),
         }
 
         // This check must happen first, so that `self.first_call_span` is set even when other
         // errors occur.  If `first_call_span` is unset, then any errors produced will be replaced
         // with the "couldn't find a call" error.
         if self.first_call_span.is_some() {
-            return report_error(call.span(), format_args!(
-                "found multiple calls to {}", self.subject_function_path.to_token_stream(),
-            ));
+            return report_error(
+                call.span(),
+                format_args!(
+                    "found multiple calls to {}",
+                    self.subject_function_path.to_token_stream(),
+                ),
+            );
```

Comment:
Please revert the whitespace changes on `Folder` - rustfmt has expanded a bunch of things that don't particularly need expanding